src/scale/format.rs

```diff
@@ -16,6 +16,7 @@
 //! Ths module defines a [`Format`], which is basically a [`StoreFormat`] and an
 //! [`OrderFormat`] and describes the different possible wire formats of a bit sequence.
 
+use alloc::string::String;
```

Comment:
Nit; missing newline:

```suggestion
use alloc::string::String;

```